pkg/cluster/majorversionupgrade.go

```diff
@@ -16,11 +16,11 @@ import (
 
 // VersionMap Map of version numbers
 var VersionMap = map[string]int{
-	"12": 120000,
```

Comment:
we might still need this line so everybody can update the operator version and manage major version upgrade via configuration (e.g. change Spilo image at a later point that lacks 12)